src/Symfony/Component/Cache/Tests/Adapter/SimpleCacheAdapterTest.php

```diff
@@ -27,4 +28,18 @@ public function createCachePool($defaultLifetime = 0)
     {
         return new SimpleCacheAdapter(new FilesystemCache(), '', $defaultLifetime);
     }
+
+    public function testValidCacheKeyWithNamespace()
+    {
+        $logger = new BufferingLogger();
```

Comment:
instead of using a logger, what about making a save + fetching after, then fail if this fetch is a miss?
that'd be more functional proof that it works to me.